bundles/org.openhab.binding.kvv/src/main/java/org/openhab/binding/kvv/internal/KVVStationHandler.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.kvv.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.thing.binding.builder.ChannelBuilder;
+import org.eclipse.smarthome.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * KVVStationHandler represents a station and holds information about the trains
+ * which will arrive soon.
+ *
+ * @author Maximilian Hess - Initial contribution
+ */
+@NonNullByDefault
+public class KVVStationHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(KVVStationHandler.class);
+
+    private final KVVStationConfig config;
+
+    public KVVStationHandler(final Thing thing) {
+        super(thing);
+        logger.info("stationhandler!");
+        this.config = getConfigAs(KVVStationConfig.class);
+    }
+
+    @Override
+    public void initialize() {
```

Comment:
You should invoke `updateStatus(ThingStatus.UNKNOWN)` at the beginning of `initialize()`. In case an unexpected exception rises, the status would be stuck to INITIALIZING.

Reply:
Solved.